### PR TITLE
Fix/materialise views

### PR DESCRIPTION
**Pull request by andrew-js-wright**

### CODE REVIEW

File: seed/views/00301_team_case_overview.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+      CONCAT(om.forename, ' ', om.surname) AS name
+    , om_type.grade_code AS grade_code
+    , w.total_cases AS total_cases
+    , wpc.available_points AS available_points
+    , wpc.total_points AS total_points
+    , wo.contracted_hours AS contracted_hours
+    , wpc.reduction_hours AS reduction_hours
+    , t.id AS id
+    , wo.id AS link_id
+  FROM app.workload_owner wo
```
Comment (ddebella) [inline seed/views/00301_team_case_overview.js]: 👍

File: seed/views/00302_ldu_case_overview.js [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-    JOIN app.team t ON t.id = tv.id
+      t.description AS name
+    , SUM(w.total_cases) AS total_cases
+    , SUM(wpc.available_points) AS available_points
+    , SUM(wpc.total_points) AS total_points
+    , SUM(wpc.contracted_hours) AS contracted_hours
+    , SUM(wpc.reduction_hours) AS reduction_hours
+    , l.id AS id
+    , t.id AS link_id
+    , COUNT_BIG(*) AS count
```
Comment (ddebella) [inline seed/views/00302_ldu_case_overview.js]: Why are we doing the count, is this used anywhere?

Comment (andrew-js-wright) [reply seed/views/00302_ldu_case_overview.js]: Good question, it's a requirement for indexed views in MS SQL. https://docs.microsoft.com/en-us/sql/relational-databases/views/create-indexed-views.

```
If GROUP BY is present, the VIEW definition must contain COUNT_BIG(*) and must not contain HAVING. These GROUP BY restrictions are applicable only to the indexed view definition. A query can use an indexed view in its execution plan even if it does not satisfy these GROUP BY restrictions. 
```

**Comment by andrew-js-wright** [comment]
✅ Approved (Can't approve a PR I opened)

**Pull request merged**